arm/Microsoft.CognitiveServices/accounts/deploy.bicep

```diff
@@ -73,20 +73,20 @@ param diagnosticEventHubName string = ''
 @description('Conditional. Subdomain name used for token-based authentication. Required if \'networkAcls\' are set.')
 param customSubDomainName string = ''
 
-@description('Optional. Whether or not public endpoint access is allowed for this account.')
+@description('Optional. Whether or not public endpoint access is allowed for this account. Should be disabled for security reasons.')
```

Comment:
As we cannot enable private endpoints by default on resources I don't think we can disable public network access by default. Without private endpoints this would mean locking access to resources completely.

Reply:
Agreed. Just following the security policy here. Any counterproposal? Should I enable it again and just keep the 'Hey you should disable this' description? :D

Reply:
Updated